.vscode/tasks.json

```diff
@@ -2,11 +2,18 @@
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
   "version": "2.0.0",
+  "presentation": {
+    "echo": false,
+    "reveal": "always",
+    "focus": false,
+    "panel": "dedicated",
+    "showReuseMessage": true
+  },
```

Comment:
```suggestion
```

Reply:
umm, i think this is supposed to go w/ a specific task:

🔗 https://code.visualstudio.com/docs/editor/tasks#_custom-tasks

Reply:
oops!

🔗 https://code.visualstudio.com/docs/editor/tasks#_global-tasks

Reply:
🔗 https://code.visualstudio.com/docs/editor/tasks-appendix